sbtc-core/src/operations/op_return/withdrawal_request.rs

```diff
@@ -185,7 +185,7 @@ pub struct WithdrawalRequestData {
 /// Construct a withdrawal request transaction
 pub fn build_withdrawal_tx(
 	wallet: &Wallet<impl BatchDatabase>,
-	broadcaster_bitcoin_private_key: BitcoinPrivateKey,
```

Comment:
Why do we no longer need the broadcaster private key?

Reply:
AH, we were using the WIF before?

Reply:
Yeah we were using the WIF to get the network, which is not always the correct approach as people don't really set correct networks intentionally when creating the WIFs.